CHANGELOG.md

```diff
@@ -3,6 +3,112 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.12.0
+
+_Aug 1, 2024_
+
+### 💵 Our commercial offering is evolving!
+
+The pro plan is receiving two new packages:
+
+- `@mui/x-tree-view-pro` (available today!)
+- `@mui/x-charts-pro` (available in the coming weeks)
+
+As always, every feature released as part of the MIT plan will remain free and MIT licensed forever.
+
+This expansion of the pro plan comes with some adjustments to our pricing strategy. Learn more about those in the [Upcoming changes to MUI X pricing in 2024](https://mui.com/blog/mui-x-sep-2024-price-update/) blog post.
+
+### Highlights
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Introduce [item reordering using drag and drop](https://mui.com/x/react-tree-view/rich-tree-view/ordering/) on the `RichTreeViewPro` component
+
+  ![Item reordering using drag and drop](https://github.com/user-attachments/assets/52329632-b7d0-4754-8561-fe26a831e2b2)
```

Comment:
It would be great to use a video here instead of a GIF for better quality.
I can record the video, let me know if that helps.

Reply:
I had a MP4 in the 1st version
I moved to a GIF to have the autoplay, but if you prefer an MP4 it's fine for me


https://github.com/user-attachments/assets/d7566834-77a9-4cdf-a4c4-00c5408c7e73

Reply:
I think the quality is fair, we can make the image smaller if we wish, which should improve perceived quality, though I think the gif autoplay is better than the video 😅

Reply:
The mp4 is not very sharp either.
I've quickly recorded one, what do you think?

https://github.com/user-attachments/assets/8594c5ba-39bc-4e80-b8de-fe0b5f7605db

Reply:
If we can make the GIF look sharp by scaling it down – I'm fine with it 👍🏻

Reply:
changed your video into a gif

Reply:
Thanks, looks better now!